spec/xcpretty/parser_spec.rb

```diff
@@ -101,6 +101,11 @@ module XCPretty
       @parser.parse(SAMPLE_COMPILE_XIB)
     end
 
+    it "parses compiling storyboards" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.